docs/deployment/dns.md

```diff
@@ -0,0 +1,68 @@
+- *.<DOMAIN> / Tenant Gateway
+- *.admin.<DOMAIN> / Admin Gateway
+
+{{% alert-note %}}
+The default value for `DOMAIN` is `uds.dev`, which is intended for development purposes only. For non-development purposes, you should override this value by specifying a value for `domain` in your `uds-config.yaml`. You can read more about overriding variables [here](https://uds.defenseunicorns.com/cli/quickstart-and-usage/#variables-and-configuration). 
+{{% /alert-note %}}
+
+You can find each Gateway resource in a dedicated namespace:
+```cli
+$ kubectl get gateway -A
+NAMESPACE              NAME             AGE
+istio-admin-gateway    admin-gateway    1h
+istio-tenant-gateway   tenant-gateway   1h
+```
+
+### Bundle Configuration
+{{% alert-note %}}
+UDS Core does not include any cloud provider specific configuration by default. Additional overrides are required to deploy UDS Core on a given provider. This section will refer to AWS, but values can be substituted as needed for other providers.
+{{% /alert-note %}}
+
+The Admin and Tenant Gateways will be each be bound to an external LoadBalancer that is exposed on TCP ports 80 and 443 by default. The Admin Gateway should be configured to use an internal facing LoadBalancer and the Tenant Gateway should be configured to use an external facing LoadBalancer. Below is an example of overrides that would accomplish this:
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-lb-config
+  description: A UDS example bundle for deploying UDS Core with external Load Balancer configuration
+  version: "0.0.1"
+  
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.27.0-upstream
+
+overrides:
+ istio-admin-gateway:
+   gateway:
+     values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internal"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"
+  istio-tenant-gateway:
+    gateway:
+      values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internet-facing"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"        
+```
+{{% alert-note %}}
+These service annotations and their values are subject to change. Please reference documentation from your cloud provider to ensure their validity.
+{{% /alert-note %}}
```

Comment:
Not sure if this is the right place or necessary, but thoughts on adding a note about static IPs to reduce effects of LBs cycling? Maybe too cloud provider specific though...

Reply:
That's great callout and a good best-practice. I'm not sure if we want to get into cloud-specific deployment best practices in this document though. Unless we disagree that most folks reading this documentation are aware of this behavior, I'm leaning towards keeping this out.

Reply:
Maybe we should just remove this callout altogether, it feels unnecessary the more I think about it.

Reply:
Yeah might be fine to just delete this note - since it's just meant to be an example anyways?